framework/db/Connection.php

```diff
@@ -337,6 +337,13 @@ class Connection extends Component
      * @see masterConfig
      */
     public $masters = [];
+
+    /**
+     * @var string the class name of the [[command]] object.
+     */
+    public $commandClass = '\yii\db\Command';
```

Comment:
Re: earlier comment about consistency. No leading slash in the class string here https://github.com/yiisoft/yii2/blob/cd87d67f34364fce025c345eebcb64be9754dc7e/framework/mail/BaseMailer.php#L76